src/ext/types/StringSearch.ts

```diff
@@ -44,6 +44,12 @@ export default YveBot.types.defineExtension('StringSearch', {
     },
     {
       transform: async (results: any, rule: IRule, bot: YveBot) => {
+        if (bot.store.get(`stringsearch.${rule.name}`)) {
```

Comment:
missing `.wait` key?

Reply:
It worked because ``bot.store.get(`stringsearch.${rule.name}`)`` returns `{ wait: true }` after first execution